working copy with alliance manager and user manager

maximenkom opened the pull request:
If leader logins he redirects to page where he can manage users of his clan
If admin logins he can manage alliances

ViacheslavKruhlov commented on src/main/java/ua/cv/tim/controller/UserController.java:
```diff
@@ -58,20 +55,18 @@ public String showAdminMainPage() {
 
 	@RequestMapping(value = "/submitUserForm",method = RequestMethod.POST, produces = {MediaType.APPLICATION_JSON_VALUE})
 	public ResponseEntity<UserDTO> submitUserForm(@RequestBody UserDTO userDTO) {
 		logger.info("UserMail {}" ,userDTO.getEmail());
```
I think better to join in one line

maximenkom replied:
this was old method, now it deleted already

ViacheslavKruhlov commented on src/main/java/ua/cv/tim/controller/UserController.java:
```diff
@@ -132,5 +127,49 @@ public ModelAndView getAllUsers2() {
+			logger.error("UserController.addAllianceMember: {}", "user is not unique");
+			return new ResponseEntity<>(HttpStatus.NOT_ACCEPTABLE);
+		}
+		try {
```
clean code: try/catch in separate method

maximenkom replied:
fixed

ViacheslavKruhlov commented on src/main/java/ua/cv/tim/dao/hibernate/UserDaoImpl.java:
```diff
@@ -43,9 +56,7 @@ public User getWithRolesById(String id) {
-        System.out.println("uuid = : " + uuid);
-
+        logger.info("UserDao.getByMail  started mail is {} uuid is {} ", mail,uuid);
         Session session = getCurrentSession();
```
I think better do not create Session instance and use getCurrentSession() method instead when create Query instance

maximenkom commented:
Added new ExceptionHandlerClass in controllers folder, now its successfully handle exceptions for addAllianceMember method, fixed mistakes pointed out by collegues
